packages/components/src/tooltip/test/index.js

```diff
@@ -276,5 +276,39 @@ describe( 'Tooltip', () => {
 			// Tooltip won't show, since the mouse has left the anchor
 			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();
 		} );
+
+		it( 'should render the shortcut display text when a string is passed as the shortcut', async () => {
+			render(
+				<Tooltip text="Help text" shortcut="shortcut text">
+					<button>Hover Me!</button>
+				</Tooltip>
+			);
+
+			const button = screen.getByRole( 'button', { name: 'Hover Me!' } );
+			button.focus();
```

Comment:
We should try to avoid imperatively `focus`sing elements.

Is the over not enough to trigger the tooltip? If focus is necessary, then I'd suggest trying to use keyboard events (ie. pressing `tab`) or even clicking to focus the `button`.

cc @tyxla and @flootr who I believe recently worked on Tooltip's unit test and debated this aspect specifically

Reply:
Thanks for the ping! 

What @ciampo said, essentially. The user can't just `focus()` - they can tab into the element, or click the element, but they can't just `.focus()`. Since we aim our tests to be as real representations of the user experience as possible, we should use actions that resemble user behavior when possible.

Reply:
Thanks for the advice.

I used `focus` it because I don't need to wait for a delay, but I understood that it is important to reproduce user actions. As with many other test cases, I changed them to test that the tooltip doesn't appear immediately after hovering and that it appears after a `TOOLTIP_DELAY`.